migration/v0.15.x-v0.16.x/upgrade-apps.md

```diff
@@ -4,6 +4,8 @@
 
 1. Run migration script: `./migration/v0.15.x-v0.16.x/migrate-apps.sh`
 
+1. Run migration script: `./migration/v0.15.x-v0.16.x/migrate-crd.sh`
+
```

Comment:
For clarity you could add a line for each script, with a short description of what they will do.